internal/app/machined/pkg/runtime/v1alpha1/platform/oracle/oracle.go

```diff
@@ -61,18 +64,38 @@ func (o *Oracle) ParseMetadata(interfaceAddresses []NetworkConfig, metadata *Met
 	}
 
 	for idx, iface := range interfaceAddresses {
-		ipv6 := iface.Ipv6SubnetCidrBlock != "" && iface.Ipv6VirtualRouterIP != ""
+		ifname := fmt.Sprintf("eth%d", idx)
 
-		if ipv6 {
+		if iface.Ipv6SubnetCidrBlock != "" && iface.Ipv6VirtualRouterIP != "" {
 			networkConfig.Operators = append(networkConfig.Operators, network.OperatorSpecSpec{
 				Operator:  network.OperatorDHCP6,
-				LinkName:  fmt.Sprintf("eth%d", idx),
+				LinkName:  ifname,
```

Comment:
just curious, are we using DHCP6 instead of static IP for a reason?

Reply:
Metadata has static IPv4 and IPv6 already (it did not have one year before)